README.md

```diff
@@ -7,3 +7,24 @@ Three tools to ease the life of Open Source developers:
 - An analytics dashboard (stalled PRs & discussions, time since last comment,
   etc.)
 
+## Command Line
+<b> ```git hub pr _```</b> <br />
+&nbsp;&nbsp; Pulls down and checkout the branch of the pr. <br />
```

Comment:
Odd characters?

Reply:
That's supposed to be a space, I'm not sure how to indent or tab on markdown files besides adding spaces. Is there a better way to go about doing this?

Reply:
How about a list:

```
- `git hub pr -` :  Description goes here.
```

Reply:
oh that looks much better! will change. thanks for the tip!